### PR TITLE
Add message_structure option support

### PR DESCRIPTION
**Pull request by d-averkiev**
Hello. According to https://github.com/zuazo/chef-handler-sns/issues/9 I've added `MessageStructure` parameter handling and write some documentation.

### CODE REVIEW

**Comment by d-averkiev** [comment]
I know about "lib/chef/handler/sns/config.rb:31:7: C: Module has too many lines. [103/100]" from rubocop. I have no idea how to get rid of it in this case :(

**Comment by zuazo** [comment]
@d-averkiev thanks for your contribution! 😃 

You can add a `.rubocop.yml` file with the following content:

```yaml
Metrics/ModuleLength:
  Max: 103
```

You can see its documentation [here](https://github.com/bbatsov/rubocop/blob/master/manual/configuration.md).

**Comment by d-averkiev** [comment]
Done. Thank you.
Seems Travis failed to install some dependencies. But I didn't touch Gemfile or .gemspec.

File: lib/chef/handler/sns/config.rb [hunk 1]
```diff
@@ -209,6 +209,23 @@ def topic_arn(arg = nil)
...
+        #
+        # @api public
+        #
+        def message_structure(arg = nil)
+          set_or_return(
+            :message_structure,
+            arg,
+            kind_of: String
+          )
+        end
```
Comment (zuazo) [inline lib/chef/handler/sns/config.rb]: Could you add some tests [here](https://github.com/zuazo/chef-handler-sns/blob/master/test/test_chef_handler_sns_config.rb#L32) for the `message_structure` method?

Please, skim-read the already existing tests. You should not have trouble understanding them even if you have never done tests before with [minitest](http://docs.seattlerb.org/minitest/). They are very verbose but it does not take long to read them once you have understood one.

Of course, do not hesitate to consult me if you have any doubts.

**Review by zuazo — CHANGES_REQUESTED** [review]
Thanks for including documentation and all 😉 Only some tests are missing.

**Comment by zuazo** [comment]
Released in [`2.1.0`](https://github.com/zuazo/chef-handler-sns/releases/tag/2.1.0).

**Pull request merged**